apps/web/pages/auth/setup/index.tsx

```diff
@@ -172,10 +175,9 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
         agreedLicenseAt: new Date(),
       },
     });
-    deploymentKey = await getDeploymentKey(prisma);
   }
 
-  const isFreeLicense = deploymentKey === "";
+  const isFreeLicense = (await getDeploymentKey(prisma)) === "";
```

Comment:
These changes are needed to fix checking correctly existing environment variable for a license key in order to create the proper record in the database automatically.